dev-lang/vlang/vlang-0.1.29.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+DESCRIPTION="Simple, fast, safe compiled language for developing maintainable software."
+HOMEPAGE="https://vlang.io"
+LICENSE="MIT"
+
+SRC_URI="https://github.com/vlang/v"
+
```

Comment:
Please try to stick to the [ebuild skeleton](https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds#How_to_create_an_ebuild) as much as possible, it makes ebuilds more readable when things are where one expects them to be.

Reply:
Yes I thought writing these in my own order might not be acceptable :)